src/app/pages/Sandbox/Editor/Workspace/Project/Statistics.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styled from 'styled-components';
+import WorkspaceSubtitle from '../WorkspaceSubtitle';
+
+type Props = {
+  likeCount: number,
+  forkCount: number,
+  viewCount: number,
+};
+
+const Count = styled.span`
+  color: white;
+  font-weight: 500;
+`;
+
+const CountDescription = styled.span`
+  color: rgba(255, 255, 255, 0.6);
+`;
+
+const CountContainer = styled.div`
+  margin: .5rem 1rem;
+  font-size: .875rem;
+`;
+
```

Comment:
It's fun. I'll try and find actual logic errors too and not just style issues :)